packages/xcm-estimate/estimator.ts

```diff
@@ -96,6 +106,11 @@ export class Estimator {
     }
   }
 
+  /**
+   * Connects to the specified chain and creates an Estimator instance.
+   * @param chainInfo - Information about the chain to connect to.
+   * @returns A promise that resolves to an Estimator instance.
+   */
   static async connect(chainInfo: ChainInfo) {
```

Comment:
Add a note that this method is solely for the convenience of creating ApiPromise internally. If a user already has an ApiPromise instance, advise the user to use the constructor instead.